identity/aziot-identityd/test/good_auth_settings.toml

```diff
@@ -4,6 +4,10 @@
 hostname = "iotedge"
 homedir = "/var/lib/aziot/identityd"
 
+# Local ID settings
```

Comment:
Please add a comment indicating that this is concatenated (suffixed) to id.name for producing the CN in the local identity's X.509 certificate.

Reply:
Done